velox/common/memory/Memory.h

```diff
@@ -46,6 +46,11 @@ DECLARE_int32(memory_usage_aggregation_interval_millis);
 namespace facebook {
 namespace velox {
 namespace memory {
+#define VELOX_MEM_LOG_PREFIX "[MEM] "
```

Comment:
nit: Would you add a comment to explain how these macros are intended to be used?

Reply:
Added. Thanks!